searx/preferences.py

```diff
@@ -292,7 +292,7 @@ def transform_form_items(self, items):
 class Preferences:
     """Validates and saves preferences to cookies"""
 
-    def __init__(self, themes: List[str], categories: List[str], engines: Dict[str, Engine], plugins: Iterable[Plugin]):
+    def __init__(self, categories: List[str], engines: Dict[str, Engine], plugins: Iterable[Plugin]):
```

Comment:
Should we really remove the theme option entirely. Maybe we want to support (external) themes one day?

I don't have a finial opinion, perhaps it is better after at all,  to remove entirely first and think about a support of (external) themes when we have a need for.

Reply:
After thinking twice .. its more clear if we remove the theme from the preferences entirely / https://github.com/searxng/searxng/pull/1153#issuecomment-1117428609

Reply:
I also agree. For now we only have the simple theme and in the future we can think about external themes / a new impl. of themes as only CSS for example...

Reply:
In the master branch, to add a new theme, a maintainer creates a directory in the `templates` and `static` directories ... and then the existing theme (the `simple` theme) is forgotten ; which make sense from the maintainer point of view.

See my-spot fork for example: 
* https://gitlab.e.foundation/e/infra/spot/-/commits/master/searx/templates/etheme
* https://spot.ecloud.global <-- the `theme` parameter doesn't work.

With this PR, we make sure only the maintained theme is available.

The remaining question: how my-spot deals with its own theme? (my-spot is an example here)
* either reuse the `templates` and `static` directories.
* either use two new directories, for example `searx/etheme/templates` and `searx/etheme/static`:
   * it is easier to backport commits (no conflicts)
   * the history of the `searx/etheme` directory is clean.

I'm not sure this is a good solution, but this PR:
* make the code more straightforward / easier to understand for someone new to the project.
* enforce the fact we won't support more than one theme.

---
note: my-spot is based of searx anyway.